UnitsNet.Tests/CustomCode/FlowTests.cs

```diff
@@ -48,5 +48,7 @@ public class FlowTests : FlowTestsBase
         protected override double DecilitersPerMinuteInOneCubicMeterPerSecond => 600000.00000;
 
         protected override double KilolitersPerMinuteInOneCubicMeterPerSecond => 60.00000;
+
+        protected override double MillionStandardCubicFeetPerDayInOneCubicMeterPerSecond => 3.058175869;
```

Comment:
Hmmm….  I’m using the same converter(s), but I think the number you’re getting is cubic meters per second in one million standard cubic feet per day (i.e. the inverse).  I thought I had double checked it before sending, and I checked it again now.  Am I doing something wrong?  The test passes with the number I have, but….

Brad.

From: Andreas Gullberg Larsen [mailto:notifications@github.com]
Sent: Tuesday, July 19, 2016 1:20 PM
To: anjdreas/UnitsNet <UnitsNet@noreply.github.com>
Cc: Bradley Plett <bplett@betamachinery.com>; Author <author@noreply.github.com>
Subject: Re: [anjdreas/UnitsNet] Added a couple of units; Signed the assemblies (#174)


In UnitsNet.Tests/CustomCode/FlowTests.cs<https://github.com/anjdreas/UnitsNet/pull/174#discussion_r71405266>:

> @@ -48,5 +48,7 @@ public class FlowTests : FlowTestsBase

>          protected override double DecilitersPerMinuteInOneCubicMeterPerSecond => 600000.00000;

>

>          protected override double KilolitersPerMinuteInOneCubicMeterPerSecond => 60.00000;

> +

> +        protected override double MillionStandardCubicFeetPerDayInOneCubicMeterPerSecond => 3.058175869;

According to this converter<http://www.traditionaloven.com/tutorials/flow-rate/convert-mil-standard-ft3-day-to-m3-cubic-meter-per-second.html> I find this should be 0.3277412847222222. Double checked here http://www.onlineconversion.com/forum/forum_1120558694.htm.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/anjdreas/UnitsNet/pull/174/files/fe7e67f7796f2aa7645cfc5a2dfaf34354afea64#r71405266>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AK7pcQ3V1zkxFp8lz3dzTrg1XC4LeMPEks5qXSNDgaJpZM4JPOdq>.